org.kde.ark.json

```diff
@@ -0,0 +1,54 @@
+{
+    "id": "org.kde.ark",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "ark",
+    "rename-icon": "ark",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=dri"
+    ],
+    "modules": [
```

Comment:
You should cleanup unneeded files, at least `/include`, `/share/man`, `*.a` and `*.la` files at first glance.

Reply:
Also `pkgconfig`

Reply:
Updated